layouts/index.html

```diff
@@ -176,8 +176,8 @@ <h2 class="text-white"><i class="fa-light fa-download"></i> Quick Local Install
                             target="_blank">advanced install page <i class="fa-solid fa-arrow-up-right"></i></a> </p>
 
                     <h3 class="mt-5 text-white">Requirements</h3>
-                    <p><strong>A.</strong> NVIDIA Pascal™ or better GPU with a <a
-                            href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 6.0 and
+                    <p><strong>A.</strong> NVIDIA Volta™ or better GPU with a <a
+                            href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 7.0 and
                             above <i class="fa-solid fa-arrow-up-right"></i></a></p>
```

Comment:
Match the wording on the release selector.
```suggestion
                    <p><strong>A.</strong> NVIDIA Volta™ or higher GPU with <a
                            href="https://developer.nvidia.com/cuda-gpus" target="_blank">compute capability 7.0+
                            <i class="fa-solid fa-arrow-up-right"></i></a></p>
```